src/components/Header/Header.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+
+import { Logo } from '../Logo/Logo.component';
+import dashboardIcon from '../../assets/icons/dashboard.svg';
+
+export const Header: React.FC = () => (
```

Comment:
remove the `: React.FC`. It adds `children` to props which is not used in this case. You can leave it without a type annotation because typescript can infer the proper type by itself.